README.md

```diff
@@ -57,9 +58,17 @@ fast-folder-size .
 
 By default the Sysinternals DU.zip is downloaded from https://download.sysinternals.com/files/DU.zip.
 
-If you need to change this, e.g. to download from an internal package repository, 
-set the **FAST_FOLDER_SIZE_DU_ZIP_LOCATION** environment variable. For example:
+If you need to change this, e.g. to download from an internal package repository
+or re-use an existing du.zip, you can set the **FAST_FOLDER_SIZE_DU_ZIP_LOCATION** environment variable.
+
+For example:
+
+```shell
+export FAST_FOLDER_SIZE_DU_ZIP_LOCATION="https://your.internal.repository/DU.zip"
+```
+
+or
 
 ```shell
-export FAST_FOLDER_SIZE_DU_ZIP_LOCATION=https://your.internal.repository/DU.zip
+export FAST_FOLDER_SIZE_DU_ZIP_LOCATION="D://download/du.zip"
```

Comment:
why double slash after the drive name? I assume we're targeting Windows in this case, so shouldn't the path separator be backslash instead?

Reply:
It's personal habit. I tested the `C://**/du.zip` and `C:\**\du.zip`, they both pass the test.

Should we add another example like:

> or
> 
> ```shell
> export FAST_FOLDER_SIZE_DU_ZIP_LOCATION="D:\download\du.zip"
> ```

or just replace "D://download/du.zip" with "D:\download\du.zip" ?